contracts/traits/psp34/psp34.rs

```diff
@@ -79,31 +120,35 @@ pub trait PSP721 {
     ///
     /// Returns `SafeTransferCheckFailed` error if `to` doesn't accept transfer.
     #[ink(message)]
-    fn transfer_from(&mut self, from: AccountId, to: AccountId, id: Id, data: Vec<u8>) -> Result<(), PSP721Error>;
+    fn transfer_from(&mut self, from: AccountId, to: AccountId, id: Id, data: Vec<u8>) -> Result<(), PSP34Error>;
+
+    /// Returns current NFT total supply.
+    #[ink(message)]
+    fn total_supply(&self) -> Balance;
```

Comment:
If you want that funciton here, then please update the proposal too

Reply:
But I think if you want to put it as required function, better to remove it at all)

Reply:
I don't know man, I want to leave it here